Software/Scratch/BrickPi3Scratch.py

```diff
@@ -196,6 +241,17 @@ def read_sensor(port):
     return return_dict
 
 
+def BP_reset():
+    '''
+    Resets the BrickPi3
+    '''
+
+    global SensorType
```

Comment:
I think yes. reset_all() sets all four sensor types to none, and it also stops the motors are returns control of the LED back to the firmware. Unless you want the motors to not stop, I think reset_all() would be best in here.